src/main/java/Main.java

```diff
@@ -0,0 +1,207 @@
+import spark.ModelAndView;
+import spark.Session;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.*;
+
+/**
+ * Created by Nigel on 7/13/16.
+ */
+public class Main {
+    public static final String SESSION_USERNAME = "userName";
+    public static final int ITEM_TOTAL = 5;
+    public static final String ALL_USER_CARS = "allUsersCars";
+    public static int idCounter = 0;
+    public static int page = 1;
+
+    static Map<String, User> users = new HashMap<>();
+    static List<Car> carsListOfAllUsers = new ArrayList<>();
+
+    public static void main(String[] args) {
+
+        Spark.init();
+
+        /**
+         * Requests to the server
+         */
+        Spark.get("/", (request, response) -> {
+
+            boolean viewNext = false;
+            boolean viewPrevious = false;
+            int nextPage = page+1;
+            int previousPage = page-1;
+
+            Session session = request.session();
+            Map templateData = new HashMap();
+
+            String userName = session.attribute(SESSION_USERNAME);
+
+            List<Car> userCars = new ArrayList<>();
+            List<Car> othersCars = new ArrayList<>();
+            List<Car> carsPage = new ArrayList<>();
+
+            /*
+             * Takes the cars that the current user created out of the list of all cars ever created. All other cars go into a different list.
+             */
+            for (Car car : carsListOfAllUsers){
+                if (car.getAuthor().equals(userName)){
+                    userCars.add(car);
+                }
+                else{
+                    othersCars.add(car);
+                }
+            }
+
+            //allows pagination over the list of cars that were created by users not currently logged in.
+            if (othersCars.size()<=5){
+                for (Car car : othersCars){
+                    carsPage.add(car);
+                }
+            }else {
```

Comment:
This else block should have logic that makes the if statement above not needed.  You should always try to make the code cover all cases.  This is important as you just hard coded 5 as a page limit in the code itself instead of at the top of the class.